### ISSUE TITLE
File Search not working

### ISSUE THREAD

**Issue by svenvNL** [jlongster/debugger.html#354]
I am running Windows 7. The problem occurs in Chrome 51.0.2704.103, Firefox latest stable (can't find version number) and Firefox Developer edition
I can't open file search. pressing ctrl + p opens the browsers print screen.

With the following Firefox add-on I changed the print keybinding: https://addons.mozilla.org/nl/firefox/addon/s3menu-wizard/
The add-on is promoted on this page: https://support.mozilla.org/en-US/kb/keyboard-shortcuts-perform-firefox-tasks-quickly

**Comment by jasonLaster** [comment]
Hey @svenvNL, this should be possible to debug by looking at [App.js](https://github.com/jlongster/debugger.html/blob/master/public/js/components/App.js#L36). The keydown handler is set there and you can easily add a breakpoint in `toggleSourcesSearch`. It might help to change the handler to open file search on another keystroke like just "p" and see if that resolves the issue. Happy to help debug further. It's also possible you're branch is not on latest...

**Comment by svenvNL** [comment]
in app,js, I changed

```javascript
toggleSourcesSearch(e) {
    if (e.metaKey && e.key == "p") {
      this.setState({ searchOn: !this.state.searchOn });
      e.preventDefault();
    }
  },
```

to

```javascript
toggleSourcesSearch(e) {
    if (e.key == "p") {
      this.setState({ searchOn: !this.state.searchOn });
      e.preventDefault();
    }
  },
```

This worked fine for me and made file search work as intended.

**Comment by jasonLaster** [comment]
Thanks @svenvNL what browser are you using? I could see this being a timing thing that might be out of our control.

**Comment by svenvNL** [comment]
I am running Chrome Canary 54.0.2799.0. The same happens in Firefox and Chrome stable.

**Comment by svenvNL** [comment]
I just looked up mdn (wich I should have done much earlier). The metakey on Windows is the WIN key, https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/metaKey. Normally, when the ⌘ key is used on a Mac, the ctrl key is used on Windows. Pressing WIN + P opens a native windows dialog to duplicate my screen to a projector.

Changing from e.metaKey to e.ctrlKey makes it work on Windows.
the code would look like this:

```javascript
  toggleSourcesSearch(e) {
    if (e.metaKey && e.key == "p" || e.ctrlKey && e.key == "p") {
      this.setState({ searchOn: !this.state.searchOn });
      e.preventDefault();
    }
  },
```

**Comment by jasonLaster** [comment]
Ahh, @svenvNL I was just about to type that this is likely a mac/windows thing. Thanks for investigating. We probably want to find a shortcut library that handles this, but that's a great fix.

**Comment by jasonLaster** [comment]
@svenvNL have you stopped by our irc room? 
https://github.com/jlongster/debugger.html/blob/master/README.md#getting-involved

Would be easy to debug in there :)

**Comment by svenvNL** [comment]
Send the quick and temp fix as a PR, #361
